flake.nix

```diff
@@ -195,7 +195,6 @@
       }
       {
         packages = inputs.std.harvest inputs.self [ "toolchain" "scripts" ];
-        library = inputs.std.harvest inputs.self [ "toolchain" "library" ];
```

Comment:
is this backwards? you got rid of lots of stuff in `scripts` not `library`?

Reply:
this was temporary and committed by mistake, we are not exposing `library` to the flake, though in a later PR I'll make a case for why we might want to do that